common/src/main/java/io/novafoundation/nova/common/data/secrets/v2/SecretStoreV2.kt

```diff
@@ -122,6 +122,16 @@ suspend fun SecretStoreV2.getAccountSecrets(
     }
 }
 
+fun AccountSecrets.ethereumKeyPair(): EncodableStruct<KeyPairSchema>? = fold(
+    left = { it[MetaAccountSecrets.EthereumKeypair] },
+    right = { it[ChainAccountSecrets.Keypair] }
+)
+
+fun AccountSecrets.substrateKeyPair(): EncodableStruct<KeyPairSchema>? = fold(
+    left = { it[MetaAccountSecrets.EthereumKeypair] },
+    right = { it[ChainAccountSecrets.Keypair] }
+)
```

Comment:
And the same here

Reply:
Answered above. Also there seems to be an error here - we are retrieving ethereum keypair for substrate secrets